_projects/use_coala.md

```diff
@@ -74,7 +75,8 @@ can be used to help select a repository that meets those criteria.
 
 To filter out repositories that do not meet the first two requirements, combine
 
-1. [`stars:>99`](https://github.com/search?o=desc&q=stars%3A%3E%3D99&ref=searchresults&s=updated&type=Repositories&utf8=%E2%9C%93), and
+1. [`stars:>99`](https://github.com/search?o=desc&q=stars%3A%3E%3D99&ref=searchresults&s=updated&type=Repositories&utf8=%E2%9C%93), 
```

Comment:
this line includes a URL, so it shouldnt need modification because of the ignore rule